students/kegan/session04/dict_lab.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+"""
+Kathryn Egan
+"""
+
+dic = {
+    'name': 'Chris',
+    'city': 'Seattle',
+    'cake': 'Chocolate'}
+
+print(dic)
+dic.pop('cake')
+print(dic)
+dic['fruit'] = 'Mango'
+print(dic)
+print('Dictionary keys: {}'.format(dic.keys()))
+print('Dictionary values: {}'.format(dic.values()))
+print('"cake" in dictionary keys: {}'.format('cake' in dic))
+print('"Mango" in dictionary values: {}'.format('Mango' in dic.values()))
+
+t_dict = {key: value.lower().count('t') for key, value in dic.items()}
+
+s2 = {i for i in range(21) if i % 2 == 0}
+s3 = {i for i in range(21) if i % 3 == 0}
+s4 = {i for i in range(21) if i % 4 == 0}
+print('s2:{}'.format(s2))
+print('s3:{}'.format(s3))
+print('s4:{}'.format(s4))
```

Comment:
when it's this simple -- not much need for the format string:

`print("s4:", s4)`